samples/components/dialogs/new text field/connection-dialog.tcd

```diff
@@ -0,0 +1,19 @@
+<connection-dialog class='kyvos'>
```

Comment:
Here, the class is 'kyvos', and below, in the tdr it's 'postgres_odbc2'. Even if this isn't a full plugin (I don't see a manifest), they should probably be aligned. Maybe use 'new_text_field_sample' to be descriptive?

Reply:
Thanks good catch.

Reply:
Interestingly it worked like this.